reference/modules/pgroonga-standby-maintainer.md

```diff
@@ -0,0 +1,59 @@
+---
+title: pgroonga_standby_maintainer module
+upper_level: ../
+---
+
+# `pgroonga_standby_maintainer` module
+
+Since 2.4.2.
+
+This is still an experimental feature.
+
+## Summary
+
+The `pgroonga_standby_maintainer` module provides executing [the `pgroonga_wal_apply()` function][pgroonga-wal-apply] and [the `pgroonga_vacuum()` function][pgroonga-vacuum] automatically on stadnby database.
+
+Normally, if we use PGroonga with streaming replication, PGroonga's WAL doesn't apply on a standby database.
+Therefore, for example, a first "SELECT" after we switch standby database to primary database may become slow.
+Because a time for applying not applicable PGroonga's WAL exist.
```

Comment:
I feel `a time ... exist(s).` is hard to read. (I'm not familiar with English, so maybe it's just me...)

How about a sentence like this?

```
Because there is time to apply not applicable PGroonga's WAL to the standby database.
```